### DISCUSSION TITLE
Shortest route

### DISCUSSION THREAD

**Discussion by ghost** [organicmaps/organicmaps#3070] [Ideas]
I mainly, due to my type of transportation, is only interested in the shortest route.
Will this be an option at a point, to e.g. choose car and shortest possible route?
Maybe by using user default and maximum speed.

**Comment by biodranik** [comment]
The current algorithm always chooses the shortest route and uses maxspeed. We plan to add alternate routes.

**Comment by Esmolante** [comment]
I think you mean the shortest in time, and  twikedk means in distance.

**Reply by biodranik** [reply]
My answer was misleading, right, I probably misunderstood you. Yes, atm the algo chooses the fastest way. The shortest one could lead through dirty/residential roads, so it should have more criteria in addition to distance.

**Reply by ghost** [reply]
I'm not sure what you mean by 'dirty residential roads'? Like unpaved?

**Reply by biodranik** [reply]
Slash was missing, sorry )

**Reply by ghost** [reply]
OK. Then dirty roads will already be 'fixed' by the 'no unpaved roads' switch and going through 'residential roads' is exactly what I want, if that gives me the shortest route.
